src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+    this.hidden = false;
+    Animal.alive.push(this);
```

Comment:
```suggestion
    Animal.alive.push(this);
```
DRY. You write this line two times: here (in Herbivore constructor) and in Carnivore constructor. Why don't you just write `Animal.alive.push(this);` inside the Animal constructor, from which these classes inherit?